src/mgr/MgrPyModule.cc

```diff
@@ -172,31 +172,43 @@ int MgrPyModule::load()
 
   // Find the class
   // TODO: let them call it what they want instead of just 'Module'
-  auto pClass = PyObject_GetAttrString(pModule, (const char*)"Module");
+  auto tempClass = PyObject_GetAttrString(pModule, (const char*)"MgrModule");
```

Comment:
yes, checking the `tp_subclasses` works. but because `tp_subclasses` is an internal member variable and only for CPython's internal usage, this is fragile. we should get  using `PyModule_GetDict()` and  check them using `PyObject_IsSubclass()`, the pseudo code looks like
```c++
auto locals = PyModule_GetDict(pModule);
PyObject *key, *value;
Py_ssize_t pos = 0;
while (PyDict_Next(locals, &pos, &key, &value)) {
  if (!PyType_Check(value)) {
    continue;
  }
  if (PyObject_IsSubclass(value, mgr_module_type)) {
    // ...
  }
}
```
where `mgr_module_type` is the`PyObject` of `MgrModule` .